extensions/typescript-language-features/src/languageFeatures/organizeImports.ts

```diff
@@ -88,13 +137,25 @@ class ImportsCodeActionProvider implements vscode.CodeActionProvider {
 	public constructor(
 		private readonly client: ITypeScriptServiceClient,
 		private readonly kind: vscode.CodeActionKind,
-		private readonly title: string,
-		private readonly sortOnly: boolean,
+		private readonly Command: OrganizeImportsCommandClass,
 		commandManager: CommandManager,
 		private readonly fileConfigManager: FileConfigurationManager,
 		telemetryReporter: TelemetryReporter,
 	) {
-		commandManager.register(new OrganizeImportsCommand(client, telemetryReporter));
+		commandManager.register(new Command(`typescript.${Command.id}`, client, telemetryReporter));
```

Comment:
Since the sort and remove commands needed to be declared in the package.json, I used `typescript.` as a prefix. But the plain Organize Imports command had an id of `_typescript.organizeImports`, so now its leading underscore is removed. Is that ok or do I need to add it back?

Reply:
The `_` just indicates that the command was private. Since they are now used in the command palette though, removing the `_` is good